composer.json

```diff
@@ -30,7 +30,8 @@
         "phpactor/tolerant-php-parser": "dev-main",
         "phpactor/map-resolver": "^1.5.0",
         "webmozart/assert": "^1.11",
-        "webmozart/path-util": "^2.3"
+        "webmozart/path-util": "^2.3",
+        "illuminate/view": "^8.83"
```

Comment:
- [ ] I refactored this out already so no need anymore I think.

Reply:
It is in fact needed as I managed to use fully compiled blade for lsp resolving using a slightly modified compiler.

Reply:
i don't really feel comfortable depending on having a hard dependency on a framework component

Reply:
Pretty sure i can copy just the needed parts over without all the rest